BUILDING.md

```diff
@@ -366,43 +366,46 @@ as `deps/icu` (You'll have: `deps/icu/source/...`)
 ## Building Node.js with FIPS-compliant OpenSSL
 
 It is possible to build Node.js with the
-[OpenSSL FIPS module](https://www.openssl.org/docs/fipsnotes.html) on POSIX
-systems. Windows is not supported.
+[OpenSSL FIPS Object Module 2.0](https://www.openssl.org/docs/fips.html) on
+POSIX systems. Windows is not supported.
 
 Building in this way does not mean the runtime is FIPS 140-2 validated, but
-rather that the runtime uses a validated module. In addition, the validation for
-the underlying module is only valid if it is deployed in accordance with its
-[security policy](http://csrc.nist.gov/groups/STM/cmvp/documents/140-1/140sp/140sp1747.pdf).
+rather that the runtime uses a validated module. In addition, the validation
+for the underlying module is only valid if it is deployed in accordance with
+the
+[OpenSSL FIPS 140-2 Security Policy](http://csrc.nist.gov/groups/STM/cmvp/documents/140-1/140sp/140sp1747.pdf).
 If you need FIPS validated cryptography it is recommended that you read both
-the [security policy](http://csrc.nist.gov/groups/STM/cmvp/documents/140-1/140sp/140sp1747.pdf)
-and [user guide](https://openssl.org/docs/fips/UserGuide-2.0.pdf).
+the Security Policy and the
+[OpenSSL FIPS Object Module 2.0 User Guide](https://openssl.org/docs/fips/UserGuide-2.0.pdf).
 
 ### Instructions
 
-1. Obtain a copy of openssl-fips-x.x.x.tar.gz.
-   To comply with the security policy you must ensure the path
-   through which you get the file complies with the requirements
-   for a "secure installation" as described in section 6.6 in
-   the [user guide](https://openssl.org/docs/fips/UserGuide-2.0.pdf).
+1. Obtain a copy of `openssl-fips-x.x.x.tar.gz`, where x.x.x is the most recent
+   version, e.g. `openssl-fips-2.0.16.tar.gz`.
+   To comply with the Security Policy you must ensure the path through which
+   you get the file complies with the requirements for a "secure
+   installation" as described in section 6.6 in the User Guide.
```

Comment:
Maybe put the user guide's link in the bottom as a reference and linkify `User Guide` here, in case people just jump straight to the `Instructions`.